service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/OthersController.java

```diff
@@ -49,7 +50,12 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(@RequestBody Cal
 
         List<Element<Other>> others = confidentialService.removeConfidentialDetails(allOthers);
 
-        caseDetails.getData().put("others", Others.from(others));
+        Others othersList = Others.from(others);
```

Comment:
not a list, var names are the wrong way round.

Reply:
updated the variable names